src/core/obj.js

```diff
@@ -703,12 +703,18 @@ var Catalog = (function CatalogClosure() {
               'app.launchURL',
               'window.open'
             ];
-            var regex = new RegExp('^(?:' + URL_OPEN_METHODS.join('|') + ')' +
-                                   '\\((?:\'|\")(\\S+)(?:\'|\")(?:,|\\))');
+            var regex = new RegExp('^\\s*(' + URL_OPEN_METHODS.join('|') + ')' +
+                                   '\\((?:\'|\")(\\S+)(?:\'|\")' +
```

Comment:
> let's replace `\\S+` to `[^\'\:]*`

Will change that, assuming of course that you meant `[^\'\"]*` since a URL ought to be able to contain a semicolon :-)